build/ci.mk

```
@@ -42,7 +42,7 @@ ci/snyk-test:
 			-v $(CURDIR):/go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-w /go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-e SNYK_TOKEN \
-			snyk/snyk:golang snyk test --severity-threshold=high
+			snyk/snyk:golang-1.15 snyk test --severity-threshold=high
```

Setting `GO111MODULE` to `auto` achieves the same and doesn't pin versions

```suggestion
			-e GO111MODULE=auto \
			snyk/snyk:golang snyk test --severity-threshold=high
```